Dockerfile

```diff
@@ -0,0 +1,13 @@
+# used JDK11 because on jdk-8 I get SUREFIRE-1588. Debian Docker images are a bit dead to me :(
+FROM maven:3.6.0-jdk-11
+LABEL maintainer="Baptiste Mathus <batmat@batmat.net>"
+
+RUN mkdir /project
+VOLUME /project/work
+ADD . /project
+WORKDIR /project
+RUN mvn clean package && \
+    cd target && \
+    mv deprecated-usage-in-plugins-*-SNAPSHOT-jar-with-dependencies.jar deprecated-usage-fat.jar
```

Comment:
that is one of the reason on change the `<finalName/>` in Maven

Reply:
I didn't check, but I suspect it's not usable anyway here, this is why I simply relied on Docker while building. AFAIK, `finalName` will apply to the usual .jar being generated, not the `...-jar-with-dependencies.jar` one.

Reply:
ok, I went through the Maven Assembly plugin docs again and you are correct. The `<finalName/>` is not used anymore. It used to be when the plugin had the goal `assembly:assembly`.